src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -2302,6 +2318,45 @@ QueryPushdownSqlTaskList(Query *query, uint64 jobId,
 }
 
 
+/*
+ * IsOuterTableOfOuterJoin tests based on the join information envoded in a
+ * RelationRestriction if the table accessed for this relation is
+ *   a) in an outer join
+ *   b) on the outer part of said join
+ *
+ * The function returns true only if both conditions above hold true
+ */
+static bool
+IsOuterTableOfOuterJoin(RelationRestriction *relationRestriction)
+{
```

Comment:
can you double check the code coverage? might just be a blip

Reply:
Sure, will check before merging.

Reply:
Ran code-cov couple of times, it's all good now.